app/generator/trail_generator.rb

```diff
@@ -0,0 +1,28 @@
+class TrailGenerator
+  def self.save_trails(data)
+    data.each do |trail|
+    saved_trail =  Trail.find_or_create_by(unique_id: trail["unique_id"]) do |t|
+        t.name        = trail["name"]
+        t.city        = trail["city"]
+        t.state       = trail["state"]
+        t.country     = trail["country"]
+        t.lat         = trail["lat"]
+        t.lng         = trail["lon"]
+        t.description = trail["description"]
+        t.directions  = trail["directions"]
+        t.unique_id   = trail["unique_id"]
+      end
```

Comment:
`end` at 14, 6 is not aligned with `saved_trail =  Trail.find_or_create_by(unique_id: trail["unique_id"]) do |t|` at 4, 4